site/style.css

```diff
@@ -166,6 +166,97 @@ h5 {
 
 /*=============================================================================*/
 
+/* News ticker */
+/* Source: https://www.cssscript.com/create-a-simple-news-ticker-using-pure-css-css3/ */
+
+.news {
+    background-color: #fefefe;
+    box-shadow: inset 0 2px 5px 0 rgba(0, 0, 0, .05), 0 2px 25px 2px rgba(0, 0, 0, .2) !important;
+    max-width: 100%;
+    height: 40px;
+    margin-left: auto;
+    margin-right: auto;
+    overflow: hidden;
+    border-radius: 10px;
+    padding: 3px;
+    -webkit-user-select: none;
+    text-align: center;
+    position: relative;
+}
+
+.news__item-list {
+    position: absolute;
+    left: 0;
+    top: 0;
+    width: 100%;
+    margin: 0;
+    padding: 0;
+    list-style: none;
+    animation-name: ticker;
+    animation-duration: 30s;
+    animation-iteration-count: infinite;
+    animation-timing-function: ease;
+    animation-fill-mode: both;
+    animation-delay: 3s;
+}
+
+.news-item {
+    line-height: 40px;
+    height: 40px;
+    padding-left: 15px;
+    padding-right: 15px;
+    display: flex;
+}
+
+.news-item__date {
+    font-weight: bold;
+    color: #19b37b;
+    min-width: fit-content;
+}
+
+.news-item__link, .news-item__link:hover {
+    color: #000;
+}
+
+.news-item__link {
+    display: block;
+    margin: 0 0 0 0.5rem;
+    font-weight: 500;
+    overflow: hidden;
+    white-space: nowrap;
+    text-overflow: ellipsis;
+    text-decoration: none;
+    -webkit-font-smoothing: antialiased;
+    -webkit-user-select: none
+}
+
+.news ul:hover { animation-play-state: paused }
+.news span:hover+ul { animation-play-state: paused }
+
+@keyframes ticker {
```

Comment:
Is it possible set to the position as percentages of the frame height rather than absolute pixels? On my phone, I notice the text is slightly higher than the centre of the frame:

![image](https://user-images.githubusercontent.com/2068515/122094165-daf4b680-cdd9-11eb-970c-0ecc03817125.png)

There is also the similar text overlap as the narrow window on desktop, but not to the same extent I guess because of the window height.

Reply:
This should be fixed now 👍

Reply:
The text overlap is no more! :tada: I still see the text slightly off the vertical centre on my phone; but I notice this doesn't happen in your screen, or in my narrow browser window, so maybe it is just me? :confused:

Reply:
Strange 🤔 What phone and browser are you using?

Reply:
I'm using a Pixel 3 with Firefox.

Reply:
I'm using a Galaxy S20 (Android 11) with Firefox (89.1.1) and I'm not having that issue 🤔
![Screenshot_20210616-091308_Firefox](https://user-images.githubusercontent.com/8475900/122226071-9cb3d180-ce83-11eb-92fe-fdd89777132b.jpg)

Reply:
I think I am just cursed :weary:

Reply:
Confirmed for cursed :imp: just checked on my partners phone which is an identical model with the same version of FF :woman_facepalming: It is so minor anyways, overall this looks great so I'm happy to approve :grin:

Reply:
That's so strange!